sdk/src/env.rs

```diff
@@ -14,6 +14,7 @@ pub mod internal {
 
 pub use crate::binary::{ArrayBinary, Binary, FixedLengthBinary};
 pub use internal::meta;
+pub use internal::require;
```

Comment:
I think this is unnecessary per https://github.com/stellar/rs-stellar-contract-sdk/pull/245.

Reply:
Ok, removed.